livingdoc-fixture-converter/src/main/java/org/livingdoc/fixture/converter/number/ByteConverter.kt

```diff
@@ -0,0 +1,6 @@
+package org.livingdoc.fixture.converter.number
+
+
+open class ByteConverter : AbstractNumberConverter<Byte>() {
```

Comment:
I was thinking the other way around. In general the examples will be written down before the fixture is written. The developer than has to decide which data type fits best. Maybe the code he is calling only takes a short?
I think we should provide converters for all of Java's datatypes. Even though I can't remember if I ever used a `Short` or `Byte` explicitly before.
But you bring up to interesting types: `BigInteger` and `BigDecimal`, we should definitely support those as well. I think I've overlooked them.
_As a general fallback the developer can always just use `Number`, that should cover all (?)_